kornia/utils/draw.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional, Union
+
+import torch
+
+
+def rectangle(
+        image: torch.Tensor,
+        boxes: torch.Tensor,
+        color: Optional[Union[float, torch.Tensor]] = 1.0,
+        fill: Optional[bool] = False) -> torch.Tensor:
+    """Draws N rectangles on a batch of image tensors
+
+    Args:
+        image (torch.Tensor): is tensor of BxCxHxW
+        boxes (torch.Tensor): represents number of rectangles to draw in BxNx4
+            N is the number of boxes to draw per batch
+            4 is in (top_left.y, top_left.x, bot_right.y, bot_right.x) ie h1, w1, h2, w2
```

Comment:
make sure we follow the same convention every where in the library. /cc @shijianjian probably we should add somewhere documentation about our convention for bounding boxes.

Reply:
I was thinking either to use [y, x, h, w] or [x1, y1, x2, y2] to represent the bounding boxes, for a more meaningful gradient. Most bounding box algorithms are using the conversion of [y, x, h, w] since it more close to a regression problem than outputting two points. That is why I think we might follow the same conversion as [y, x, h, w].

Reply:
@shijianjian why [y,x] instead of [x,y]?